deploy/clickhouse-keeper/clickhouse-keeper-1-node-256M-for-test-only.yaml

```diff
@@ -0,0 +1,246 @@
+              {
+                echo "<yandex><keeper_server>"
+                echo "<server_id>${MY_ID}</server_id>"
+                echo "<raft_configuration>"
+                for (( i=1; i<=$SERVERS; i++ )); do
+                    echo "<server><id>${i}</id><hostname>$NAME-$((i-1)).${DOMAIN}</hostname><port>${RAFT_PORT}</port></server>"
+                done
+                echo "</raft_configuration>"
+                echo "</keeper_server></yandex>"
+              } >> /tmp/clickhouse-keeper/config.d/generated-keeper-settings.xml &&
+              cat /tmp/clickhouse-keeper/config.d/generated-keeper-settings.xml && 
+              clickhouse keeper --config-file=/etc/clickhouse-keeper/keeper_config.xml
+          readinessProbe:
+            exec:
+              command:
+                - bash
+                - -xc
+                - "OK=$(bash -c 'exec 3<>/dev/tcp/127.0.0.1/2181 && printf \"ruok\" >&3 && timeout 5 cat <&3');
+                   if [[ \"$OK\" == \"imok\" ]];
+                   then
+                      STATE=$(bash -c 'exec 3<>/dev/tcp/127.0.0.1/2181 && printf \"mntr\" >&3 && timeout 5 cat <&3' | grep zk_server_state | cut -d \" \" -f 2);
+                      if [[ \"$STATE\" == \"leader\" ]]; then
+                        SYNCED_FOLLOWERS=$(bash -c 'exec 3<>/dev/tcp/127.0.0.1/2181 && printf \"mntr\" >&3 && timeout 5 cat <&3' | grep zk_synced_followers | cut -d \" \" -f 2 | cut -d \".\" -f 1);
+                        if [[ $SYNCED_FOLLOWERS == $(( $SERVERS - 1 )) ]]; then
+                          exit 0;
+                        else
+                          exit 1;
+                        fi;
+                      elif [[ \"$STATE\" == \"follower\" ]]; then
+                        PEER_STATE=$(bash -c 'exec 3<>/dev/tcp/127.0.0.1/2181 && printf \"mntr\" >&3 && timeout 5 cat <&3' | grep zk_peer_state);
+                        if [[ \"$PEER_STATE\" == \"following - broadcast\" ]]; then
+                          exit 0;
+                        else
+                          exit 1;
+                        fi;
+                      fi;
+                    else
+                      exit 1;
+                    fi"
+            initialDelaySeconds: 15
+            timeoutSeconds: 15
+        - name: zookeeper-exporter
+          imagePullPolicy: IfNotPresent
+          image: dabealu/zookeeper-exporter:latest
+          resources:
+            requests:
+              memory: "64M"
+              cpu: "0.3"
+            limits:
+              memory: "128M"
+              cpu: "1"
+          command:
+            - zookeeper-exporter
+            - "-listen"
+            - 0.0.0.0:7000
+            - "-zk-hosts"
+            - 127.0.0.1:2181
+          livenessProbe:
+            exec:
+              command:
+                - sh
+                - -c
+                - "OK=$(echo ruok | nc 127.0.0.1 2181); if [[ \"$OK\" == \"imok\" ]]; then exit 0; else exit 1; fi"
```

Comment:
that exporter - may be just check that 7000 is opened?

Reply:
No! This is the proper place of liveness probe. 
It allows us to properly monitor `up` metrics for /metrtics endpoint from prometehus side, when keeper down then exporter also shall down, otherwise prometheus will continue scraping exporter and will always get metrics (look like it's  a just specified zabbix-exporter behavior)

Reply:
OK. But how to detect the situation when the keeper is up, but the exporter is down / can't start?

Reply:
We already will catch this corner case, when `keeper` is up, but `zookeer-exporter` is stuck, we will have `up == 0` on `prometheus` side because this metric calculates on `prometheus` side not inside `zookeeper-exporter`

Reply:
But from k8 side it will look as 'everything is ok'. OK, I hope that exporter will not stuck often :)